src/backend/partaj/core/email.py

```diff
@@ -363,3 +363,42 @@ def send_validation_requested(cls, validation_request, activity):
         }
 
         cls.send(data)
+
+    @classmethod
+    def send_referral_closed(cls, contact, referral):
+        """
+        Send the "referral closed" email to the user who just closed the referral.
+        """
+
+        template_id = settings.SENDINBLUE["REFERRAL_CLOSED_TEMPLATE_ID"]
+
+        data = {
+            "params": {"case_number": referral.id},
+            "replyTo": cls.reply_to,
+            "templateId": template_id,
+            "to": [{"email": contact.email}],
+        }
+
+        cls.send(data)
+
+    @classmethod
+    def send_referral_changeurgencylevel(
+        cls, contact, referral, newurgency, oldurgency
+    ):
+        """
+        Send the "referral changeurgencylevel" email to all stakeholders.
+        """
+        template_id = settings.SENDINBLUE["REFERRAL_CHANGEURGENCYLEVEL_TEMPLATE_ID"]
+
+        data = {
+            "params": {
```

Comment:
I think this email should include the new due date along with the old due date. If we just include urgency levels it might not be as useful for users